pkgs/dart_tooling_mcp_server/README.md

```diff
@@ -27,13 +27,31 @@ WIP. This package is still experimental and is likely to evolve quickly.
 
 ## Usage
 
-To use this package, you will need to compile the `bin/main.dart` script to exe
-and use the compiled path as the command in your MCP server config.
+This server only supports the STDIO transport mechanism and runs locally on
+your machine. Many of the tools require that your MCP client has `roots`
+support, and usage of the tools is scoped to only these directories.
 
-```shell
+The server entrypoint lives at `bin/main.dart`, and is can be ran however you
+choose, but the easiest way is to run it as a globally activated package:
+
+```sh
+dart pub global activate -s path .
+```
+
+And the, assuming the pub cache bin dir  is set up on your PATH, the
```

Comment:
```suggestion
And then, assuming the pub cache bin dir is set up on your PATH, the
```

Reply:
can you add instructions for adding the pub cache bin dir to your path?

Reply:
done